taxi/forms.py

```diff
@@ -0,0 +1,51 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
+from django.views import View
+
+from taxi.models import Driver, Car
+
+
+class LicenseNumberValidatorMixin(forms.ModelForm):
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) == 8:
+            RegexValidator(
+                regex=r"[A-Z]{3}\d{5}",

```

Comment:
The regex pattern `[A-Z]{3}\d{5}` should match the description in the error message. Ensure that the pattern and the message are consistent. If the pattern is correct, adjust the error message to reflect this.